wrappers/python/indy/error.py

```diff
@@ -165,13 +166,17 @@ class ErrorCode(IntEnum):
 
     # Operation is not supported for payment method
     PaymentOperationNotSupportedError = 704,
-    
+
     # Extra funds on inputs
     PaymentExtraFundsError = 705
 
 
 class IndyError(Exception):
     # error_code: ErrorCode
+    # message: Optional[str]
```

Comment:
Only backtrace should be optional from my point of view. Also it is good to add note how to get backtrace in comments.

Reply:
And i suggest to rename backtrace to indy_backtrace

Reply:
> Only backtrace should be optional from my point of view. Also it is good to add note how to get backtrace in comments.

I have put message and backtrace as optional fields to save backward compatibility (init IndyError with error code only).